app/assets/javascripts/discourse/app/templates/mobile/modal/create-account.hbs

```diff
@@ -0,0 +1,177 @@
+{{#create-account email=accountEmail disabled=submitDisabled action=(action "createAccount")}}
+  {{#unless complete}}
+    {{plugin-outlet name="create-account-before-modal-body"}}
+    {{#d-modal-body class=modalBodyClasses}}
+
+      {{#if skipConfirmation}}
+        {{loading-spinner size="large"}}
+      {{/if}}
+
+      {{#if showCreateForm}}
+        <div class="create-account-form">
+          <div class="login-welcome-header">
+            <h1>Welcome!</h1> <img src="/images/emoji/twitter/wave/{{skinToneId}}.png" alt="" class="waving-hand">
```

Comment:
welcome is never translated?

Reply:
The image URL won't work with subfolder and the `emoji_set` site setting be used instead of hardcoding `twitter`?

Reply:
this has been fixed